apidoc/Modules/CloudPush/CloudPush.yml

```diff
@@ -231,12 +231,22 @@ events:
         Touching it launches the app, and fires this event.
         
         This is only applicable if you have set [showTrayNotification](Modules.CloudPush.showTrayNotification) to true.
+    properties:
+      - name: payload
+        summary: A JSON string of your data payload. Use JSON.parse to turn this in to an object you can use.
+        type: String
```

Comment:
Is this part of a new module version? If so, we should add the Titanium version, that includes the updated module, e.g. `since: 5.4.0`.